course_catalog/etl/youtube_test.py

```diff
@@ -101,32 +100,31 @@ def mock_video_batch_data(mock_video_data):
     return data
 
 
-@pytest.fixture
-def mock_channel_data():
-    """Mock video channel yml"""
-    return """
----
-channels:
-  - offered_by: MIT
-    channel_id: channel1
-    playlists:
-      - playlist1
-  - offered_by: OCW
-    channel_id: channel2
-    playlists:
-      - playlist2
-"""
+def mock_channel_file(offered_by):
+    """Mock video channel github file"""
+
+    content = """
```

Comment:
You can change this to an [f-string](https://realpython.com/python-f-strings/#multiline-f-strings) to avoid the `format()`:

```python
content = f""""
offered_by: {offered_by}
"""
```